README.md

```diff
@@ -4,6 +4,8 @@
 
 # Gateway
 
+**`v2` branch is not yet stable and recommended for production use.**
```

Comment:
While it's kind of understandable from the context, I think it would be safer with another "not" – "not recommended for production use". Otherwise somebody can understand it differently :)